y_lb.libraries.yml

```diff
@@ -9,6 +9,8 @@ main:
   css:
     theme:
       assets/css/colors.css: { minified: false }
+      assets/css/typography.css: { minified: true }
```

Comment:
Always bump version when you add/remove anything from library 
![image](https://user-images.githubusercontent.com/563412/218503768-6b3bf0ce-cc18-4ae8-af0b-f1f53b44092b.png)

Reply:
@podarok thank you! Fixed!